library/packages/src/modules/SlideShow.rb

```diff
@@ -139,6 +139,7 @@ def main
       @language = "en"
       @widgets_created = false
       @user_switched_to_details = false
+      @user_switched_to_release_notes = false
```

Comment:
to be honest I do not like this "flags" they are exclusive ( details and release notes ) and they are also related.
What about having just one variable `@user_switched_to` with values `:details`, `:slides` and `release_notes`? so we know where user want to be?